### ISSUE TITLE
 Error: kafka server: Topic with this name already exists.

### ISSUE THREAD

**Issue by ivialex-mcd** [Mongey/terraform-provider-kafka#190]
I created Kafka cluster (AWS MSK) in my first pipeline. So, in another pipeline I run to create topics. But when I run again my pipeline show this error

```
 Error: kafka server: Topic with this name already exists.

   on main.tf line 70, in resource "kafka_topic" "rfmkafkatopic_dlq":
   70: resource "kafka_topic" "rfmkafkatopic_dlq" {
```

Does anyone know how I can solve it?

**Comment by emmaroberts-nbs** [comment]
Yeah, ACLs turned out to be the root cause of me hitting this issue to.

Has anyone figured out what the ACL should be to allow an msk user (using sasl/scram auth) to have full access?

**Comment by Constantin07** [comment]
> Has anyone figured out what the ACL should be to allow an msk user (using sasl/scram auth) to have full access?

I know how ACL needs to look like for TLS based authentication ..

**Comment by ivialex-mcd** [comment]
Hi guys,

My apologies for my delayed answer. I discover my issue. I was running my terraform sripts using GitHub Runner. And GitHub Runner don't have access to private Subnet in AWS. 

So, I setup self-hosted runner (EC2 Instance) and changed my github workflows, then works for me. 

And for my local machine I created a cliend VPN endpoint to work for me.

Thanks everyone.
